garak/detectors/encoding.py

```diff
@@ -4,56 +4,66 @@
 output by the model
 """
 
+from typing import Iterable
 from garak import _config
 from garak.attempt import Attempt
 from garak.detectors.base import Detector, StringDetector
 
 
 class DecodeMatch(StringDetector):
-    """Is there a direct hash decoding?"""
+    """Is there a direct decoding of anything in attempt["notes"]["triggers"]?"""
 
-    bcp47 = "en"
+    bcp47 = "*"
     matchtype = "str"
 
-    def __init__(self, config_root=_config):
-        import garak.probes.encoding
+    def __init__(self, config_root=...):
```

Comment:
Default value should still be `_config`.
```suggestion
    def __init__(self, config_root=...):
```